x-pack/plugins/security_solution/public/detections/components/rules/rule_execution_status/index.tsx

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
There are couple of places in rule details page, where execution status is used as well. Didn't want to drag those possible changes to this PR, related to tables.
@elastic/security-detections-response-rules, if it makes sense I can create a ticket to reuse this component if this implementation is accepted and  it's fine by everyone.